_posts/2017-01-27-welcome-to-vital-literature-review.markdown

```diff
@@ -1,6 +1,43 @@
 ---
-layout: post
-title:  "Welcome to the Literature Review site of VITAL."
-date:   2017-01-27 12:18:56 -0500
+layout:    post
+title:     Welcome to the Literature Review site of VITAL
+date:      2017-01-27 12:18:56 -0500
+permalink: /welcome-how-to/
 ---
-On this site, you'll find all the essential papers any deep learning expert should read.
\ No newline at end of file
+
+## Goal of this website
+
+The goal of this website is to collaborate by helping each other stay up to date with new advances. It will also help people that are new to the field to acquire essential knowledge about the state of the art.
+
+## How to add a review
+
+The process for adding reviews is _git-centric_. Basically, **you just need to add a file to the repo and make a pull request**. Let's go into the details :
+
+1. Clone [the `vitalab.github.io` repo](https://github.com/vitalab/vitalab.github.io) on your computer.
+2. Create the file `YYYY-MM-DD-title-of-your-review.markdown` in the folder `/_posts`. `name-of-your-review` can be the title of the paper, for example. **The format is important, the name of the file must begin by the date of creation of your review.**
```

Comment:
... your review.** For exemple, `2017-01-29-You Only Look Once Unified Real-Time Object Detection.markdown` would be a valid name.